examples/example.rs

```diff
@@ -13,16 +23,73 @@ struct Foo {
     // Or you can set the default
     #[builder(default = 20)]
     z: i32,
+
+    // #[builder(default)] without parameter - don't require this field
+    // #[builder(setter(strip_collection))] without parameter - start with the default and extend from there
+    #[builder(default, setter(strip_collection))]
+    v0: Vec<i32>,
+
+    // No `default`: This field must be set at least once.
+    // You can explicitly create the collection from the first item (but this is not required even without `default`).
+    #[builder(setter(strip_collection(from_first = |first| vec![first])))]
```

Comment:
Is `from_first` without arguments even meaningful? What would be the default?

As for `from_iter` - I think maybe it should be **the** default - as in, without an argument you won't even need to specify it. Instead of keeping the actual target collection we can keep `I: Iterator`, and each time we extend with the builder, we can make an [`std::iter::Chain`](https://doc.rust-lang.org/std/iter/struct.Chain.html). We can still specify `from_iter = |it| ...` if `FromIterator` is not available for that collection or if for whatever reason we need custom behavior.